abstracts_presentations/ams_am_2017/abstract.rst

```diff
@@ -2,23 +2,23 @@ Building a roadmap for a mid-sized community software project: Perspectives from
 =======================================================================================
 *Scott Collis, Jonathan Helmus and Cory Weber*
 
-The Python ARM Radar Toolkit (Py-ART, [1]) is an architecture for interacting
+The Python ARM Radar Toolkit (Py-ART, [1]) is an environment for interacting
 with and extracting insight from weather radar data. While it is difficult to
 track total usage we estimate Py-ART has over a hundred active users with
-thousands of downloads from GitHub, pip and Conda installs. With fifteen
-developers and a very diverse funding background, led by the Department of
-Energy's ARM program, it is important to have a vision for how development on
-the package will proceed over the next five years. The codebase must serve the
+thousands of downloads from GitHub, pip and conda installs.  Py-ART has fifteen
+contributors and a very diverse funding background, led by the Department of
```

Comment:
I like "contributor" rather than "developer".  The latter conjures the idea of active, ongoing and funded work on the software.